pkg/hub/submarineragent/manifests/operator/submariner.io-submariners-cr.yaml

```diff
@@ -15,6 +15,7 @@ spec:
   ceIPSecPSK: {{ .IPSecPSK }}
   clusterCIDR: ""
   globalCIDR: "{{ .GlobalCIDR }}"
+  loadBalancerEnabled: "{{ .LoadBalancerEnabled }}"
```

Comment:
Not familiar with addon code, QQ: does this change support setting loadbalancer mode per cluster?

Reply:
yes this is the `Submariner` CR - this matches the exiting field in the [CRD](https://github.com/submariner-io/submariner-operator/blob/9bf9b0ba4eb131c2ad8c4c7b9005150cc46d662d/api/v1alpha1/submariner_types.go#L62).

Reply:
Ack, I wasn't sure how we get  {{ .LoadBalancerEnabled }} value, IIUC it's passesd at  [1] in brokerInfo parameter.
So, wanted to make sure we can set it per cluster 


[1]
https://github.com/stolostron/submariner-addon/blob/main/pkg/hub/submarineragent/controller.go#L514

Reply:
`LoadBalancerEnabled` comes from the [SubmarinerConfig CR](https://github.com/stolostron/submariner-addon/blob/d1b28298d6dc3a53bc11469d7a84ea359f7ba20c/pkg/hub/submarinerbrokerinfo/brokerinfo.go#L186) which is per managed cluster.